coderd/users_test.go

```diff
@@ -393,10 +393,16 @@ func TestNotifyUserStatusChanged(t *testing.T) {
 		for _, expected := range expectedNotifications {
 			found := false
 			for _, sent := range notifyEnq.Sent() {
+				userData, userDataOk := sent.Data["user"].(map[string]any)
```

Comment:
Nit: rather use `require.IsType` so we get specific failures. 

What you have here will be difficult to debug given the number of clauses.